site/content/docs/5.3/migration.md

```diff
@@ -49,6 +49,7 @@ If you need more details about the changes, please refer to the [v5.3.2 release]
     <summary><span class="badge bg-success">New</span> CSS variables:</summary>
     <ul>
       <li><code>--bs-accordion-btn-hover-bg</code></li>
+      <li><code>--bs-alert-icon-bg</code></li>
```

Comment:
This CSS var doesn't exist. This PR now only uses `--bs-alert-icon-bg` which was already there if I'm not mistaken.